### ISSUE TITLE
Skip mame bios/devices/chd files

### ISSUE THREAD

**Issue by lodx-xd** [mmatyas/pegasus-frontend#857]
I wonder if it is possible to get the same result as the #363 but using Retroarch on Windows? I've been racking my brain for a time already on how to prevent the bios/devices/chd files from appearing in the game list, but to no avail.

As retroarch needs all these files in the same folder as the roms, when pegasus scans the folder they appear along with the games, I thought it would be something simple case of removing the entry for these files from the "metadata.pegasus.txt" file, but it seems to be more complex than that apparently.

I just started this whole "business" of configuring frontends by myself, sorry beforehand if my question is too ignorant.

**Comment by lodx-xd** [comment]
Ok, disabling "Logiqx" (not sure exactly what this option does) in the "Gaming" options made the chd files no longer show up in the games list, but bios/devices still show up.

**Comment by lodx-xd** [comment]
Thanks for the replay, I added the line you said in the metadata file but it didn't work, the chd files still keep showing up in the games list inside Pegasus.

I don't know what could be wrong, here is my metadata file in case you are interested in seeing it https://pastebin.com/MgqCmAsJ

I also tried to use the line `ignore-file: neogeo.zip` to see if I could remove the neo geo bios and the devices too from the games list but I was also unsuccessful.

**Comment by mmatyas** [comment]
Hm the file seems fine, maybe they are detected from a different source? Could you try disabling entries under _Settings -> Enable/disable data sources_?

**Comment by lodx-xd** [comment]
The problem was the data sources, I had re-enable `Logiqx`, with `Logiqx` enable no matter what modifications I make using the `ignore-` line, the ignored extensasions/files will still show up in the list inside Pegasus.

Another thing is that just ignoring the files/extensions is not enough, I need to ignore and remove their entries in the metadata too, which is a pain, because so far I haven't found a way to make Skraper not scan multi-disc games to prevent it to include their entry in the metadata file or download the media files =/

Apart from this inconvenience, I can now ignore the games and extensions I want.

Thank you for taking the time to help me.

**Comment by mmatyas** [comment]
Yeah, the Logiqx support will read all game entries in its DAT file, and the metadata file will add all `game` entries as well. I guess this is a Skraper problem then... Will close the issue for now, but feel free to report any other problems that come up!
